### ISSUE TITLE
I Am Bad At Titles. Version 2.2b

### ISSUE THREAD

**Issue by Mattabase** [s0cks/RemoteIO#11]
https://dl.dropboxusercontent.com/u/15990261/ShareX/2015/03/crash-2015-03-15_14.26.17-client.txt

**Comment by s0cks** [comment]
XD, I will look into this as well thank you :)

**Comment by Mattabase** [comment]
Lazy Matt Is Lazy. https://dl.dropboxusercontent.com/u/15990261/ShareX/2015/03/crash-2015-03-15_14.59.14-server.txt

**Comment by s0cks** [comment]
XD soo many crashes, this is good though gives me something to do :)

**Comment by s0cks** [comment]
Fixed the first one

**Comment by Mattabase** [comment]
More bad at this, the remote interface handheld thingy does not open when you click in the air.

**Comment by Mattabase** [comment]
Also more of me being bad at reports. https://dl.dropboxusercontent.com/u/15990261/ShareX/2015/03/crash-2015-03-18_18.28.54-server.txt

**Comment by s0cks** [comment]
Yeah the Remote Accessor doesn't open when you right click the air, its intentional, as when you right click any block with it it acts as though you right clicked the original block

**Comment by Mattabase** [comment]
This is the only thing in my config. 

# Configuration file

optout {
    # Opt-out of localization updates, and only use lang files packaged with the JAR
    B:localization_update=false
}

**Comment by Mattabase** [comment]
Seems wrong
